source/opt/function.h

```diff
@@ -60,6 +61,10 @@ class Function {
   // Appends a basic block to this function.
   inline void AddBasicBlock(std::unique_ptr<BasicBlock> b);
 
+  // Adds a basic block to a function after |insert_point|.
+  inline void AddBasicBlock(const ir::BasicBlock* insert_point,
```

Comment:
make it take an iterator and return the new iterator (so everything keep being valid)